File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1074.cpp
#include <iostream>

using namespace std;

int main(){
    int n, i;
    long int x;
    string resposta;
    cin >> n;
    for(i=0;i<n;i++){
        cin >> x;
        resposta="";
        if(x==0){
            cout << "NULL";
        } else{
            if(x%2==0){
                resposta+="EVEN ";
            } else{
                resposta+="ODD ";
            }
            if(x>0){
                resposta+="POSITIVE";
            } else{
                resposta+="NEGATIVE";
            }
        }
        cout << resposta << endl;
    }

    return 0;
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1074.cs
using System;

class URI
{
    static void Main() {
        int n = int.Parse(System.Console.ReadLine().Trim()), x;

        for(int i=0;i<n;i++){
            x = int.Parse(System.Console.ReadLine().Trim());
            if(x==0){
                Console.Write("NULL\n");
            } else{
                if(x%2==0){
                    Console.Write("EVEN ");
                } else{
                    Console.Write("ODD ");
                }
                if(x>0){
                    Console.Write("POSITIVE\n");
                } else{
                    Console.Write("NEGATIVE\n");
                }
            }
        }
    }
}